.github/workflows/validate.yml

```diff
@@ -13,11 +13,14 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - name: install deps
+        run: |
+          sudo apt -q update
+          sudo apt -q install libseccomp-dev
```

Comment:
Hmm, this job only runs golangci-lint. I can't see how it needs `libseccomp-dev` package now

Reply:
We now do a CGO import within `runc` and `golangci-lint` will give an error on the new `patchbpf` package. I only added this because this particular job was failing as a result.